### ISSUE TITLE
remove as much lodash as possible

### ISSUE THREAD

**Issue by jwetzell** [jwetzell/showbridge#635]
- [ ] `cloneDeep`
- [ ] `has`
- [ ] `get`
- [ ] `memoize`
- [ ] `template`
- [ ] `merge`
- [ ] `isEqual`
- [ ] `orderBy`
- [ ] `noop`
- [ ] `sortBy`

**Comment by tm1000** [comment]
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore

**Comment by jwetzell** [comment]
yeah, I had looked through that when I wrote up this issue. Most of it seems to be just replace lodash with basically the same code that `lodash` is doing on the inside so not sure that improves anything. For the most part these functions, besides the obviously dumb `noop`, don't have good analogues IMO in native JS.
